src/libs/actions/PolicyConnections.ts

```diff
@@ -0,0 +1,55 @@
+import Onyx from 'react-native-onyx';
+import type {OnyxUpdate} from 'react-native-onyx';
+import * as API from '@libs/API';
+import type {OpenPolicyAccountingPageParams} from '@libs/API/parameters';
+import {READ_COMMANDS} from '@libs/API/types';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+function openPolicyAccountingPage(policyID: string) {
+    const policyKey = `${ONYXKEYS.COLLECTION.POLICY}${policyID}` as const;
+
+    const optimisticData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: policyKey,
+            value: {
+                isLoadingConnections: true,
+                hasConnectionsDataBeenFetched: false,
```

Comment:
`isLoadingConnections` and `hasConnectionsDataBeenFetched` seem to have exactly the opposite values, I think you can just have one of them.

Reply:
you're right. we only need `hasConnectionsDataBeenFetched`